File: gluecodium/src/test/resources/smoke/defaults/output/android/com/example/smoke/BlobDefaults.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class BlobDefaults {
    @NonNull
    public byte[] emptyList;
    @NonNull
    public byte[] deadBeef;
    public BlobDefaults() {
        this.emptyList = new byte[] {  };
        this.deadBeef = new byte[] { (byte) 222, (byte) 173, (byte) 190, (byte) 239 };
    }
}

File: gluecodium/src/test/resources/smoke/defaults/output/android-kotlin/com/example/smoke/BlobDefaults.kt
/*

 *
 */

@file:JvmName("BlobDefaults")

package com.example.smoke


class BlobDefaults {
    @JvmField var emptyList: ByteArray
    @JvmField var deadBeef: ByteArray



    constructor() {
        this.emptyList = byteArrayOf(  )
        this.deadBeef = byteArrayOf( 222.toByte(), 173.toByte(), 190.toByte(), 239.toByte() )
    }





}

